src/js/common/schemaform/validation.js

```diff
@@ -96,6 +96,16 @@ export function transformErrors(errors, uiSchema) {
  *
  * The function is passed errors, fieldData, formData, and otherData and
```

Comment:
The function is passed errors, fieldData, pageData, formData, and otherData (wha..?)